app/javascript/shared/components/tags.jsx

```diff
@@ -374,6 +374,7 @@ class Tags extends Component {
       listing,
       fieldClassName,
       onFocus,
+      pattern,
```

Comment:
This might be unnecessary duplication, but I think having finer-grain control might be good (based on the last attempt working for articles but not listings). Happy to take this out and include the pattern only in the component if this seems like too much.